### ISSUE TITLE
Add missing lang attribute to html

### ISSUE THREAD

**Issue by lucianmurmurache** [activist-org/activist#703]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

This is an accessibility (and seo) fix/improvement, currently the html element does not have a [lang attribute](https://www.w3.org/WAI/standards-guidelines/act/rules/b5c3f8/).

Nuxt seems to have a nice [plugin ](https://nuxt.com/docs/guide/directory-structure/plugins) feature where the attribute can simply be injected into each page by default, without having to add it manually to all <Head>...</Head> elements so I will opt for this approach.

**Comment by andrewtavis** [comment]
Thanks for opening the issue, @lucianmurmurache! Will review the PR in the coming days :) :)

**Comment by andrewtavis** [comment]
Will close this once I send along the period for the comment, @lucianmurmurache. Again no stress on that 😊

**Comment by andrewtavis** [comment]
Sent it along c90b8a0 so we can close this, @lucianmurmurache :) Thanks again for all the work here!
